src/Utils/time.ts

```diff
@@ -1,46 +1,49 @@
-import { metaphysics } from "./metaphysics"
+import { graphql } from "react-relay"
+import { Environment, fetchQuery } from "relay-runtime"
 
 const getLocalTimestampInMilliSeconds = () => {
   return Date.now()
 }
 
-interface SystemTimeResults {
-  data: {
-    system: {
-      time: {
-        unix: number
+export async function getOffsetBetweenGravityClock(
+  relayEnvironment: Environment
+): Promise<number> {
+  interface SystemTimeResults {
```

Comment:
No need to define this manually, just use the typing emitted by relay-compiler as normal.